src/core/dev/cq_mgr_rx_regrq.cpp

```diff
@@ -86,9 +86,9 @@ cq_mgr_rx_regrq::~cq_mgr_rx_regrq()
 
 mem_buf_desc_t *cq_mgr_rx_regrq::poll(enum buff_status_e &status)
 {
-    mem_buf_desc_t *buff = NULL;
+    mem_buf_desc_t *buff = nullptr;
 
-    if (unlikely(NULL == m_rx_hot_buffer)) {
+    if (unlikely(nullptr == m_rx_hot_buffer)) {
```

Comment:
Please be consistent with changing if statements.
sometimes its x == null and sometimes its !x

Reply:
change in all places

Reply:
OK